test/algorithms/test_is_valid.hpp

```diff
@@ -98,17 +97,19 @@ struct is_convertible_to_closed<Ring, bg::ring_tag, bg::open>
 template <typename Polygon>
 struct is_convertible_to_closed<Polygon, bg::polygon_tag, bg::open>
 {
-    typedef typename bg::ring_type<Polygon>::type ring_type;
+    using ring_type = typename bg::ring_type<Polygon>::type;
 
     template <typename InteriorRings>
     static inline
     bool apply_to_interior_rings(InteriorRings const& interior_rings)
     {
-        return bg::detail::check_iterator_range
-            <
-                is_convertible_to_closed<ring_type>
-            >::apply(boost::begin(interior_rings),
-                     boost::end(interior_rings));
+        auto const end = boost::end(interior_rings);
+        return std::find_if(
+            boost::begin(interior_rings), end, 
+            []( auto const& ring ){ 
+                return ! is_convertible_to_closed<ring_type>::apply(ring);
+            }
+        ) == end;
```

Comment:
Is this style your personal preference or is this formatting caused by the 80 characters limit?

Reply:
personal preference but see in the comment above for an alternative

Reply:
what about 
```
return std::find_if(boost::begin(interior_rings), end, 
    []( auto const& ring ){ 
        return ! is_convertible_to_closed<ring_type>::apply(ring);
    }) == end;
```